server/model/step.go

```diff
@@ -67,3 +68,13 @@ func (p *Step) Running() bool {
 func (p *Step) Failing() bool {
 	return p.Failure == FailureFail && (p.State == StatusError || p.State == StatusKilled || p.State == StatusFailure)
 }
+
+type StepType uint8 //	@name StepType
+
+const (
```

Comment:
Could we re-use the pipeline model in here.

Reply:
what do you want to reuse?

Reply:
The enum from pipeline

Reply:
Ah ... no because of imports

Reply:
e.g. just exec:
```sh
go list -f '{{.Imports}}' github.com/woodpecker-ci/woodpecker/pipeline/backend/... | tr -d '[]' | tr ' ' '\n' | grep 'githu
b.com/woodpecker-ci/woodpecker' | sort | uniq
```